website/docs/r/serverless_instance.html.markdown

```diff
@@ -35,6 +36,7 @@ resource "mongodbatlas_serverless_instance" "test" {
 * `provider_settings_provider_name` - (Required) Cloud service provider that applies to the provisioned the serverless instance.
 * `provider_settings_region_name` - (Required) 	
   Human-readable label that identifies the physical location of your MongoDB serverless instance. The region you choose can affect network latency for clients accessing your databases.
+* `continuous_backup_enabled` - (Optional) Flag that indicates whether the serverless instance uses Serverless Continuous Backup. If this parameter is false, the serverless instance uses Basic Backup. 
```

Comment:
Basic is fine for data source but let's include some links here to explain these backup types: 
```suggestion
* `continuous_backup_enabled` - (Optional) Flag that indicates whether the serverless instance uses [Serverless Continuous Backup](https://www.mongodb.com/docs/atlas/configure-serverless-backup). If this parameter is false or not used, the serverless instance uses [Basic Backup](https://www.mongodb.com/docs/atlas/configure-serverless-backup). 
```

Reply:
Added additional information